boards/posix/native_posix/hw_models_top.c

```diff
@@ -26,20 +30,28 @@ static u64_t end_of_time = NEVER; /* When will this device stop */
 /* List of HW model timers: */
 extern u64_t hw_timer_timer; /* When should this timer_model be called */
 extern u64_t irq_ctrl_timer;
+#ifdef CONFIG_HAS_SDL
+extern u64_t sdl_event_timer;
+#else
```

Comment:
Better instead ifdef the code below that is only used for SDL to not affect other use cases.